api/all/src/main/java/io/opentelemetry/api/baggage/propagation/Element.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.api.baggage.propagation;
+
+import java.util.BitSet;
+
+class Element {
+
+  private final BitSet excluded = new BitSet(128);
+
+  private boolean leadingSpace;
+  private boolean readingValue;
+  private boolean trailingSpace;
+  private int start;
+  private int end;
+  private String value;
+
+  Element(char[] excludedChars) {
+    for (char separator : excludedChars) {
```

Comment:
odd calling this loop variable `separator`. It's not always a separator is it?

Reply:
Right, forgot to change this after refactoring.